src/callgraph_mode.ml

```diff
@@ -0,0 +1,24 @@
+open! Core
+
+type t =
+  | Last_branch_record
+  | Frame_pointers
+  | Dwarf
+[@@deriving sexp]
+
+let param =
+  let open Command.Param in
+  flag
+    "-callgraph-mode"
+    (optional (Command.Arg_type.create (fun str -> t_of_sexp (Sexp.of_string str))))
```

Comment:
We should also allow people to specify short-forms here too, i.e. "lbr", "fp", "dwarf", to be familiar to people used to `perf`. If the sexp parsing fails, we can fallback to a lookup table.